Developed in-house, not fed back.
Solaris-specific: configuration file customization.

--- doc/config/lighttpd.annotated.conf.orig	2025-03-22 21:40:18.000000000 +0200
+++ doc/config/lighttpd.annotated.conf	2025-03-31 22:21:35.397612326 +0300
@@ -13,11 +13,11 @@
 ## if you add a variable here. Add the corresponding variable in the
 ## chroot example as well.
 ##
-var.log_root    = "/var/log/lighttpd"
-var.server_root = "/srv/www"
-var.state_dir   = "/run"
-var.home_dir    = "/var/lib/lighttpd"
-var.conf_dir    = "/etc/lighttpd"
+var.log_root    = "/var/lighttpd/1.4/logs"
+var.server_root = "/var/lighttpd/1.4"
+var.state_dir   = "log_root"
+var.home_dir    = "server_root"
+var.conf_dir    = "/etc/lighttpd/1.4"
 
 ##
 ## run the server chrooted.
@@ -58,7 +58,7 @@
 ## used in:
 ## conf.d/deflate.conf
 ##
-var.cache_dir   = "/var/cache/lighttpd"
+var.cache_dir   = server_root + "/cache"
 
 ##
 ## Base directory for sockets.
@@ -97,8 +97,8 @@
 ## Run as a different username/groupname.
 ## This requires root permissions during startup.
 ##
-server.username  = "lighttpd"
-server.groupname = "lighttpd"
+server.username  = "webservd"
+server.groupname = "webservd"
 
 ##
 ## Enable lighttpd to serve requests on sockets received from systemd
@@ -114,7 +114,7 @@
 ##
 ## Document root
 ##
-server.document-root = server_root + "/htdocs"
+server.document-root = server_root + "/docroot"
 
 ##
 ## The value for the "Server:" response field.
@@ -126,7 +126,7 @@
 ##
 ## store a pid file
 ##
-server.pid-file = state_dir + "/lighttpd.pid"
+server.pid-file = state_dir + "/run.pid"
 
 ##
 #######################################################################
@@ -180,7 +180,7 @@
 ## poll   (recommended if none of above are available)
 ## select (*not* recommended)
 ##
-#server.event-handler = "epoll"
+server.event-handler = "solaris-devpoll"
 
 ##
 ## The basic network interface for all platforms at the syscalls read()
@@ -327,7 +327,7 @@
 ##                                 "index.htm", "default.htm" )
 ##
 index-file.names += (
-  "index.xhtml", "index.html", "index.htm", "default.htm", "index.php"
+  "index.php", "index.html", "index.htm", "default.htm", "index.xhtml"
 )
 
 ##
@@ -374,7 +374,7 @@
 ## Format: <errorfile-prefix><status-code>.html
 ## -> ..../status-404.html for 'File not found'
 ##
-#server.errorfile-prefix    = server_root + "/htdocs/errors/status-"
+#server.errorfile-prefix    = "/var/lighttpd/1.4/errors/status-"
 
 ##
 ## mimetype mapping
--- doc/config/conf.d/fastcgi.conf.~1~	2025-03-22 21:40:18.000000000 +0200
+++ doc/config/conf.d/fastcgi.conf	2025-04-01 08:57:15.857249205 +0300
@@ -22,8 +22,8 @@
 #fastcgi.server = ( ".php" =>
 #                   ( "php-local" =>
 #                     (
-#                       "socket" => socket_dir + "/php-fastcgi-1.socket",
-#                       "bin-path" => server_root + "/cgi-bin/php5",
+#                       "socket" => socket_dir + "/php-fastcgi.socket",
+#                       "bin-path" => "/usr/php/bin/php-cgi",
 #                       "max-procs" => 1,
 #                       "broken-scriptfilename" => "enable",
 #                     ),
